src/codec2_math.h

```diff
@@ -31,8 +31,7 @@ extern "C"
 {
 #endif // __cplusplus
 
-// Forward declare COMP struct (use comp.h to refer to items inside the struct).
-struct COMP;
+#include "comp.h"
```

Comment:
Wouldn't this already be included in theory by ofdm.c (even if only implicitly)? Though it probably wouldn't hurt to just include it anyway.

Reply:
I feel it's good practice to include data types used by a header file.

Reply:
That works.